### PR TITLE
BREAKING feature: preparing for Three.js r167

### PR DESCRIPTION
**Pull request by 0b5vr**
This PR adapts usage of WebGPU stuff to the r167 way.

import WebGPU / NodeMaterial stuff from `three/webgpu`.
See: https://github.com/mrdoob/three.js/pull/28650

~~Three.js r167 is not released yet!~~
~~I tested the behavior by using `yarn link` on `three` and `@types/three`.~~

~~importmaps for Three.js in webgpu examples are temporarily replaced to node_modules, please change them back before merging.~~

This PR also bumps Three.js to r167.
Several changes are made even outside the WebGPU scope in order to follow r167 changes.

MToonNodeMaterial now requires r167 or higher.
The main WebGL module should still work in r137-r166.

### CODE REVIEW

**Comment by 0b5vr** [comment]
The failing test says `Cannot find module 'three/webgpu' or its corresponding type declarations.`.
It's an expected result, should be fixed once we import r167

**Comment by lin72h** [comment]
r167 just released! looking forward to 3.0 release

File: packages/three-vrm-materials-mtoon/src/nodes/MToonLightingModel.ts [hunk 1]
```diff
@@ -79,47 +79,52 @@ export class MToonLightingModel extends Nodes.LightingModel {
...
           .add(matcap)
           .mul(rimMultiply)
-          .mul(Nodes.mix(Nodes.vec3(0.0), Nodes.BRDF_Lambert({ diffuseColor: lightColor }), rimLightingMix)),
+          .mul(THREE.mix(THREE.vec3(0.0), THREE.BRDF_Lambert({ diffuseColor: lightColor }), rimLightingMix)),
       ),
     );
   }
 
-  indirectDiffuse({ irradiance, reflectedLight }: Nodes.LightingModelIndirectInput) {
+  indirect(context: THREE.LightingModelIndirectInput) {
```
Comment (yue4u) [inline packages/three-vrm-materials-mtoon/src/nodes/MToonLightingModel.ts:102]: not mentioned in https://github.com/mrdoob/three.js/wiki/Migration-Guide#166--r167 ..

**Review by yue4u — APPROVED** [review]
So we will rename exports in another PR?

File: packages/three-vrm-materials-mtoon/src/nodes/MToonLightingModel.ts (hunk 1 above)
Comment (0b5vr) [reply packages/three-vrm-materials-mtoon/src/nodes/MToonLightingModel.ts:102]: Three.js devs don't assume users to extend `NodeMaterial` 😅

**Comment by 0b5vr** [comment]
> So we will rename exports in another PR?

No, we're going to go `@pixiv/three-vrm/nodes` as is instead of renaming it to `@pixiv/three-vrm/webgpu`.
Naming it `@pixiv/three-vrm/webgpu` makes users misunderstand that the export includes the entire library with NodeMaterial stuff, considering how Three.js exports `three/webgpu`.

**Pull request merged**